src/libs/ReportActionsUtils.ts

```diff
@@ -524,12 +568,24 @@ function filterOutDeprecatedReportActions(reportActions: ReportActions | null):
  * to ensure they will always be displayed in the same order (in case multiple actions have the same timestamp).
  * This is all handled with getSortedReportActions() which is used by several other methods to keep the code DRY.
  */
-function getSortedReportActionsForDisplay(reportActions: ReportActions | ReportAction[] | null, shouldMarkTheFirstItemAsNewest = false): ReportAction[] {
-    const filteredReportActions = Object.entries(reportActions ?? {})
-        .filter(([key, reportAction]) => shouldReportActionBeVisible(reportAction, key))
-        .map((entry) => entry[1]);
+function getSortedReportActionsForDisplay(reportActions: ReportActions | null | ReportAction[], shouldIncludeInvisibleActions = false): ReportAction[] {
+    let filteredReportActions: ReportAction[] = [];
+    if (!reportActions) {
+        return [];
+    }
+
+    if (shouldIncludeInvisibleActions) {
+        filteredReportActions = Object.values(reportActions).filter(
+            (action): action is ReportAction => !(action?.originalMessage as OriginalMessageActionableMentionWhisper['originalMessage'])?.resolution,
```

Comment:
This type cast seems weird. Do we have some `filteredReportActions` which is not a `ReportAction`?

Reply:
Before we sorted out the whisper fix, I ran into some actions that were pretty much empty. I'm on it, trying to replicate the issue now
If I can't find the same cases, I'll remove the filter. It needs to be double-checked as it occurred with a specific account

Reply:
I see this has been removed. Thanks!